### PR TITLE
Add test Files to Develpment

### PR DESCRIPTION
**Pull request by elassis**
# What we have Accomplished 🏁 :
- Test the website React Components✔️
- Use Jest and React Test Library to test components ✔️
## Built With 🔨 👍 :
- React
- Jest
## Built By 🤝 : 
- @elassis 
- @alvp01

### CODE REVIEW

File: math/src/tests/calculate.test.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import calculate from "../logic/calculate"
+
+test('getting values from calculate',()=>{
+  const objresult = calculate({
+    total:'5',
+    next: '3',
+    operation:'+'
+  },'=');
```
Comment (henatan99) [inline math/src/tests/calculate.test.js]: - [ ] Nice work here but you are supposed to test all the scenarios of the `calculate` functionality. Other operations like subtraction, division, multiplication, modulus, negation,... should be tested as well. Kindly add more tests. :play:

File: math/src/tests/operate.test.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import operate from "../logic/operate"
+
+test('getting values from operate',()=>{
+  const result = operate('5','3','+');
+
+  expect((result)).toBe('8');
```
Comment (henatan99) [inline math/src/tests/operate.test.js]: - [ ]  Kindly test all the `operate` conditions.

File: math/src/tests/Calculator.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculator from '../components/calculator';
+
+it('renders correctly', () => {
+  const tree = renderer
+    .create(<Calculator />)
+    .toJSON();
+  expect(tree).toMatchSnapshot();
+});
```
Comment (henatan99) [inline math/src/tests/Calculator.test.js]: - [ ] It is great to have a snapshot test for the Calculator component. Similarly, make a simple snapshot test for each of the other components. :pray: 
- [ ] In addition, kindly add tests to simulate user interaction at least in the `MathContainer` component, using React Testing Library.

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @elassis, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
- [ ] Your project structure is faulty and I couldn't run it from the root directory (react-repo). Instead, I have to go inside and install and run from `math` directory for your app to work. I think you shouldn't alter the structure of a react app initialized by `create-react-app` command. It is like you nested one react project in another with duplicate `package.json` and other files. Kindly fix this app structure issue. I strongly suggest that you make the `math` directory the project/root directory. :pray: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉 🎉 

## Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alvp01** [comment]
> # Congratulationstada tada
> ## Hi Team,
> 
> Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
> Congratulations! tada
> ### Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you Delice!

**Pull request merged**